operator/pom.xml

```diff
@@ -33,7 +33,6 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <quarkus.operator.sdk.version>4.0.1</quarkus.operator.sdk.version>
-        <quarkus.version>2.12.2.Final</quarkus.version>
```

Comment:
@abstractj Thanks. I talked with @vmuzikar and it seems we can not align the versions between dist and operator. The operator is an extension from quarkiverse and as such has a different release cycle.

So the best is to keep this one as is.